test/recipes/15-test_ec.t

```diff
@@ -24,7 +24,7 @@ ok(run(test(["ectest"])), "running ectest");
 
 SKIP: {
     skip "Skipping EC conversion test", 3
-        if disabled("ec");
+        if disabled("ec") || disabled('deprecated-3.0');
```

Comment:
In addition to skipping this stanza, shouldn't we provide the same tests using `pkey`?

Reply:
I'm of two minds on this. This recipe is primarily about the ec command, or so I thought. Isn't there another recipe that tests the pkey command?

Reply:
I've duplicated this stanza of tests, using `pkey` the second time around.

Reply:
@levitte I think it makes sense to have them in this test suite for now, while ec still exists, so that normally we test that the deprecated and replacement command do work.

We can then proceed to move all of this into a test_pkey/etc test once we are actually removing the ec command.

Reply:
Will we remember to do that in 5+ years?
(hypothetically, 5+ years from now, "we" may be a different team, so "remember" also involves passing the plan on)

Reply:
We could add a comment about "this should be moved there once ec is removed", when ec is removed this file will cause build errors and whomever is removing ec will find the comment and can decide to ignore it completely because it does not make sense anymore ;P

Reply:
Only the first SKIP block will need removing when _ec_ is finally deleted.  I'd hope that anyone fixing things would at least look at the remaining tests and not touch them.

I don't see a pressing reason for a `test_pkey` move.  This one is called `test_ec` and that's what it is testing -- sure some uses the _ec_ command and the rest use _pkey_ but it is all _ec_ related.  Keeping the broad areas separate will help keep the noise down.

Reply:
I had to look back at `test/README` to remind myself of the group numbers, and sure enough, this one is in the "individual asymmetric cipher algorithms" group, not "openssl commands" (which is 20-24).

So, er, never mind, but in that case, I'm starting to think that all commands used in this file could be changed to "pkey" ones.  Not duplicated, replaced.

Reply:
We'd lose the testing of the _ec_ command if this were done.

Reply:
Well, here's a relevant question Emilia asked a few years ago: are we testing [this] or are we testing [that]. Here, translated to "are we testing the EC algorithm, or the EC command?" The underlying principle is that separate should test one aspect, not more.
We have been sloppy with that, but question is if we should keep being sloppy or if we should fix the sloppiness as we go. Or should we ditch that principle?

Reply:
We're testing EC algorithms here.  It is done via the command line using both of the two available commands.  This seems reasonable to me.

Might they be better in the 20-24 range?  Quite possibly.  Almost all of the 15- series tests are similar to this one in nature, if this moves they all should.

Would they be better separated into pkey and ec tests?  I'm less sure about that.  A pkey command test is going to be big and produce a lot of output, many parts of which aren't related.  The two commands used here are very closely related.

A refactoring of these tests into other places seems well beyond the scope of this PR.  An issue could be raised noting that we ought to go back and reconsider things.

Reply:
I would be satisfied with a todo comment.

Also, considering we're moving to evp for everything, this test group seems less relevant for testing algorithms per se, since we have evp_test. 

We should probably have a 20-test_pkey.t, specifically to test `openssl pkey`. That's for another PR, though

Reply:
Okay, done.